src/main/webapp/app/exercises/shared/exercise-scores/exercise-scores.component.html

```diff
@@ -210,7 +210,7 @@ <h5 jhiTranslate="artemisApp.exercise.export.options"></h5>
                         </span>
                     </ng-template>
                     <ng-template ngx-datatable-cell-template let-value="value">
-                        <jhi-result *ngIf="value.results?.[0]" [exercise]="exercise" [result]="value.results?.[0]" [participation]="value"></jhi-result>
+                        <jhi-updating-result [exercise]="exercise" [participation]="value" [showUngradedResults]="true"></jhi-updating-result>
```

Comment:
I would not use the updating result component here, since this activates a new websocket subscription for every particapton in the list (typically 50). Instructors can and should use the refresh button to load the new results.

Reply:
Thank you for the advice! I just adjust it to jhi-result. 👍